ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -15,6 +15,7 @@
 
 #define MODULE_INVOLVED_IN_ARM3
 #include <mm/ARM3/miarm.h>
+#include <fltkernel.h>
```

Comment:
Why is the FS mini-filter header here?
I don't see any code that uses\needs it.

Reply:
We can also include it globally, but it is full of stuff that we don't need and I don't want to include it everywhere (while the practice of using a single global header makes things easier, it is also questionable, whether that is "the right way/place" in general). We can duplicate the few useful macros somewhere (maybe NDK?) or create our own utility header. Suggestions?

Reply:
Ahh yes, I hadn't noticed this macro was limited to the flt header. That'll be why I only ever see it in the FS minifilter samples :)

Anyway, in this case, I'd just add the Add2Ptr macro locally in the file, wrapped in an '#ifndef Add2Ptr'

Reply:
I added the defines to ntoskrnl.h